### PR TITLE
Assignment 3 Done 

### PR DESCRIPTION
**Pull request by gunjanvjangra**
Click Counter Code and Functioning

### CODE REVIEW

**Comment by gunjanvjangra** [comment]
Google Drive Link for the functioning video: 
https://drive.google.com/file/d/1eddP_H9_r7rW6Dd1bywtOVQTwQTUCokf/view?usp=sharing

**Comment by gunjanvjangra** [comment]
Task 2 : Clone and Additional Details of Provided Feed Page 
Done

File: Assignment 2/Click Counter.mp4
Comment (i-am-SnehaChauhan) [inline Assignment 2/Click Counter.mp4:1]: The counter page is not Visible. Please provide the code and the screenshots for the same

File: Assignment 1/Question.js
Comment (i-am-SnehaChauhan) [inline Assignment 1/Question.js:1]: Late Submission

**Review by i-am-SnehaChauhan — CHANGES_REQUESTED** [review]
Add Counter page code and screenshots.

**Review by i-am-SnehaChauhan — APPROVED** [review]

**Comment by gunjanvjangra** [comment]
Assignment 3 : Counter using functional and class component + Form to fill basic details 
Done

**Pull request closed without merging**